orttraining/orttraining/training_ops/cuda/math/div_grad_impl.cu

```diff
@@ -478,7 +478,7 @@ void ImplDivGrad(
           *fdm_output_strides,
           db_output_data,
           N);
-  } else if (a_padded_strides && a_padded_strides->Size()) {
+  } else if (a_padded_strides && b_padded_strides && a_padded_strides->Size()) {
```

Comment:
avoid deferencing b_padded_strides if b_padded_strides is NULL.

Reply:
I don't quite understand the code ....

Reply:
@edgchen1 , would you like to take over it?

Reply:
ok sure

Reply:
Thank you!